log.capnp

```diff
@@ -1949,6 +1949,13 @@ struct Event {
     can @5 :List(CanData);
     controlsState @7 :ControlsState;
     sensorEvents @11 :List(SensorEventData);
```

Comment:
add the DEPRECATED suffix and move it down with the other deprecated events
```suggestion
    sensorEventsDEPRECATED @11 :List(SensorEventData);
```

Reply:
done